core/services/functions/request.go

```diff
@@ -1,20 +1,25 @@
 package functions
 
 const (
-	LocationInline     = 0
-	LocationRemote     = 1
-	LocationDONHosted  = 2
+	LocationInline    = 0
+	LocationRemote    = 1
+	LocationDONHosted = 2
+
 	LanguageJavaScript = 0
+
+	RequestStatePending       = 1
+	RequestStateComplete      = 2
+	RequestStateInternalError = 3
 )
 
 type RequestFlags [32]byte
 
 type OffchainRequest struct {
-	RequestId         []byte      `json:"requestId"`
-	RequestInitiator  []byte      `json:"requestInitiator"`
-	SubscriptionId    uint64      `json:"subscriptionId"`
-	SubscriptionOwner []byte      `json:"subscriptionOwner"`
```

Comment:
Would it make sense to keep this in and keep it set as 0x0 for Offchain responses?
Because eventually with Offchain Request -> Onchain Response this may be useful

Reply:
I agree, thanks.